packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -19,41 +19,47 @@ type Props = {
 
 const LOCALE_SWITCHER_LABEL_ID = 'locale-switcher-label';
 
-export const SingleValue = (props: CustomSingleValueProps) => (
-  <div
-    css={css`
-      flex: 1;
-      align-items: center;
-      display: flex;
-    `}
-  >
-    <WorldIcon size="big" />
-    <span
+export const SingleValue = (props: CustomSingleValueProps) => {
+  const { themedValue } = useTheme();
+  return (
+    <div
       css={css`
-        margin-left: 2px;
         flex: 1;
-        color: ${designTokens.colorAccent};
-      `}
-    >
-      {props.children}
-    </span>
-    <span
-      css={css`
-        width: 22px;
-        height: 22px;
-        border-radius: 100%;
-        background: ${designTokens.colorAccent40};
-        color: ${designTokens.colorSurface};
-        font-size: 0.9rem;
-        display: flex;
-        justify-content: center;
         align-items: center;
+        display: flex;
       `}
     >
-      {props.localeCount}
-    </span>
-  </div>
-);
+      <WorldIcon size="big" />
+      <span
+        css={css`
+          margin-left: 2px;
+          flex: 1;
+          color: ${designTokens.colorAccent};
+        `}
+      >
+        {props.children}
+      </span>
+      <span
+        css={css`
+          width: ${themedValue('22px', '26px')};
+          height: ${themedValue('22px', '26px')};
+          border-radius: 100%;
+          background: ${themedValue(
+            designTokens.colorAccent40,
+            designTokens.colorNeutral
+          )};
+          color: ${designTokens.colorSurface};
+          font-size: ${themedValue('0.9rem', '1rem')};
```

Comment:
Normal font size

Reply:
Could we maybe use here `ui-kit` tokens?
```suggestion
          font-size: ${themedValue(designTokens.fontSize15, designTokens30)};
```

Reply:
Ah thanks! https://github.com/commercetools/merchant-center-application-kit/pull/2993/commits/379af3a8b91f8004f5d44a44a9c7266b53e2051d